website/docs/developer/research/research-bamboo-integration-tests.md

```diff
@@ -0,0 +1,135 @@
+```
+Variables in terraform.tfvars can be overwritten by the `-var` field as seen above. 
+
+Some of the sensitive bamboo variables such as `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `RDS_USER_ACCESS_SECRET_ARN` have been encrypted using bamboo encryption service. While running the pipeline, those variables have to be replaced manually with the real values since bamboo does not automatically decrypt the values while running the pipeline. Make sure all sensitive variables are encrypted before pushing the changes to the repo. 
+
+The bamboo spec for running the two scripts under tasks is shown below.
+
+```yaml
+#job for deploying cumulus and orca
+Deploy RDS cluster:
+  key: DRC
+  other:
+    clean-working-dir: true
+    # Some plugin configurations are not supported by YAML Specs
+  docker:
+    image: amazonlinux:2
+    volumes:
+      ${bamboo.working.directory}: ${bamboo.working.directory}
+      ${bamboo.tmp.directory}: ${bamboo.tmp.directory}
+  tasks:
+  - checkout:
+      force-clean-build: 'true'
+      description: Checkout Default Repository
+
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash
+        chmod +x bin/deployment-rds.sh
+        bin/deployment-rds.sh
+#job for deploying cumulus and orca
+Deploy cumulus and orca:
+  key: DCO
+  other:
+    clean-working-dir: true
+    # Some plugin configurations are not supported by YAML Specs
+  docker:
+    image: amazonlinux:2
+    volumes:
+      ${bamboo.working.directory}: ${bamboo.working.directory}
+      ${bamboo.tmp.directory}: ${bamboo.tmp.directory}
+  tasks:
+  - checkout:
+      force-clean-build: 'true'
+      description: Checkout Default Repository
+
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash
+        chmod +x bin/deployment-cumulus-orca.sh
+        bin/deployment-cumulus-orca.sh
+```
+
+## Future directions
+
+The first step will be to finish deploying all cumulus tf modules via Bamboo into sandbox account. Once that is completed, automation scripts for running the integrations tests need to be created. The original plan `ORCA Integrator` in bamboo will need to be updated with the same changes made in `prototype-demo` plan's `feature/ORCA-test-bamboo` branch so that the `develop` branch is updated. In addition, older resources should be deleted from AWS once tests are validated. 
+
+The details of the DockerFile as well as the scripts currently used by GHRC to deploy their repository to Bamboo CI/CD can be found [here](https://github.com/ghrcdaac/GHRC-TF-CICD) which will be useful to look into later on. Moreover, the environment variables in bamboo can be exported using env files to better manage terraform variables needed for deployment. See [ORCA-393](https://bugs.earthdata.nasa.gov/browse/ORCA-393) for more information on this.
+
+
+Some of the cards created to finish the task include:
+- https://bugs.earthdata.nasa.gov/browse/ORCA-388
```

Comment:
May want to put a brief summary of each card so we do not have to click each link to look them up.

Reply:
added summary to each card